File: 프로그래머스/unrated/181834. l로 만들기/l로 만들기.java
class Solution {
    public String solution(String myString) {
       String answer = "";
        for (int i = 0; i < myString.length(); i++) {
            char c = myString.charAt(i);
            if (c < 'l') {
                answer += 'l';
            } else {
                answer += c;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181834. l로 만들기/l로 만들기.js
function solution(myString) {
   let charArray = myString.split('');

    for (let i = 0; i < charArray.length; i++) {
        if (charArray[i] < 'l') {
            charArray[i] = 'l';
        }
    }

    return charArray.join('');
}